src/test/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("The "+ getName() + " began its work.");
+
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("The "+ getName() + " has finished its work.");
```

Comment:
Similarly, the message in the 'stopWork' method should be more informative. It should clearly indicate that it is the Bulldozer that has stopped working. For example, "Bulldozer has finished its work."